ci/cold_96_00z.yaml

```diff
@@ -0,0 +1,20 @@
+environment:
+  icdir: /scratch1/NCEPDEV/global/glopara/data/ICSDIR/C96C48
```

Comment:
This path is going to be different on different machines. You likely should parameterize all but the last bit and add the definitions to ${machine}.sh

Reply:
Oh yes, I have not generated these yet, I planned on doing that in another PR.

Reply:
Another note, this moved because I updated yaml to use **icsdir** and forgot to check it in with the update.